CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+# Contributing
+
+Contributing to this repository, we follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within the project.
+
+## Issue tracker
+
+Whether you are looking for an opportunity to contribute or you found a bug and already know how to solve it, please go to the [issue tracker](https://github.com/ansible-collections/community.mysql/issues).
+There you can find feature ideas to implement, reports about bugs to solve, or submit an issue to discuss your idea before implementing it which can help choose a right direction initially and potentially save a lot of time and effort.
+Also somebody can already start discussing or working on implementing the same or similar idea,
+so you can cooperate to create a better solution together.
+
+## Open pull requests
+
+Look through currently [open pull requests](https://github.com/ansible-collections/community.mysql/pulls).
+You can help by reviewing them. There can be good pull requests which are not merged only because there are a lack of reviews. Also there can be worth, reviewed, but abandoned ones, which you could politely ask the original authors to complete yourself.
+And it is always worth saying that good reviews are often more valuable than pull requests themselves.
+
+## Looking for an idea to implement
+
+First, see the paragraphs above.
+
+If you came up with a new feature, it is always worth creating an issue
+before starting to write code to discuss the idea with the community first.
+
+## Step-by-step guide how to get into development quickly.
```

Comment:
@aminvakil thanks for the feedback!
The purpose of the guide (as I see it) to allow newcomers to understand a necessary minimum of what they actually should, set up a basic environment, and get their patches submitted quickly. That's why I put the link to the dev guide to the end and not put any links to the sections (like about integration tests) intentionally not to overwhelm people with not really crucial information at least for the moment.
When they get their first PR merged (I hope quickly), it's very inspiring itself (as it was at least for me) and can motivate them very well to go further. Otherwise, going through a ton of guides at this *early* stage, they can say "Oh, it's too overwhelming and time consuming to read this stuff from cover to cover, I'd better go to play a computer game":)
So it's necessary minimal given, I went through the guide myself on a clear environment - seems workable:)

Reply:
Thanks!:)